### PR TITLE
Use `pkgquery -Sif "%N"` instead of `pkgquery -S --qdepends -f "%n"`

### PR DESCRIPTION
**Pull request by larchunix**
I didn't see any regression regarding #39.

### CODE REVIEW

**Comment by f2404** [comment]
@larchunix I'm sorry but this doesn't seem to resolve the ffmpeg upgrade issue I've referenced in #39 - the command produced empty output for libvidstab.so which means that the "required by -" message will appear:
```$ package-query -Sif %N libvidstab.so```

**Comment by larchunix** [comment]
@f2404 In the code, the name of the package is used so you have to use `vid.stab` instead of `libvidstab.so`: 

    $ package-query -S --qdepends "%n" vid.stab
    openshot
    $ package-query -Sif "%N" vid.stab
    ffmpeg openshot

**Comment by f2404** [comment]
You are right, my bad :(

File: src/lib/abs.sh.in [hunk 1]
```diff
@@ -214,21 +214,11 @@ format_update() {
 			pkgver=$rversion
 			local requiredbypkg=""
 
-			local pkg_dep_on=( $(pkgquery -S --qdepends -f "%n" "$pkgname") )
+			local pkg_dep_on=( $(pkgquery -Sif "%N" "$pkgname") )
```
Comment (f2404) [inline src/lib/abs.sh.in]: Suggest adding the -1 option - this will reduce the execution time.

**Pull request merged**